bin/lib/run_from_docker

```diff
@@ -11,14 +11,19 @@ if [[ -z "${CIVIFORM_CLOUD_DEPLOYMENT_VERSION}" ]]; then
   exit 1
 fi
 
+if [[ -z "${CIVIFORM_VERSION}" ]]; then
+  echo "CIVIFORM_VERSION needs to be set to version from https://github.com/civiform/civiform/releases"
```

Comment:
We can also use "latest". Would good to point it out as it's useful for stagings.

Reply:
Updated the docs in the example config